ZitiDesktopEdge.Client/ServiceClient/AbstractClient.cs

```diff
@@ -242,4 +248,21 @@ public abstract class AbstractClient {
             }
         }
     }
+
+    public class ShouldSerializeContractResolver : DefaultContractResolver {

```

Comment:
does `[JsonIgnore]` not work??? https://www.newtonsoft.com/json/help/html/PropertyJsonIgnore.htm

Reply:
It may ignore it in both serialization and deserialization flow. I saw Jeremy using this field somewhere in the UI. So I added the customer resolver to ignore this property only in the Add Identity Message.